iceoryx_examples/icecubetray/config_and_run_icecubetray.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+WORKSPACE=$(git rev-parse --show-toplevel)
+CONFIG="OFF"
+RUN="OFF"
+SESSION=icecubetray
+tmux="tmux -2 -q"
+
+while (( "$#" )); do
+  case "$1" in
+    config)
+        CONFIG="ON"
+        shift 1
+        ;;
+    run)
+        RUN="ON"
+        shift 1
+        ;;
+    *)
+        echo "Invalid argument '$1'"
+        exit -1
+        ;;
+  esac
+done
+
+if [ "$CONFIG" == "ON" ] ; then
+    # Create groups
+    sudo groupadd -f privileged
+    sudo groupadd -f unprivileged
+    sudo groupadd -f iceoryx
+
+    # Create users w/o homedir
+    sudo useradd -M perception
+    sudo useradd -M infotainment
+    sudo useradd -M roudi
+    sudo useradd -M notallowed
+
+    # Assign users to group and disable login
+    sudo usermod -L perception -a -G privileged
+    sudo usermod -L infotainment -a -G unprivileged
+    sudo usermod -L perception -a -G iceoryx
+    sudo usermod -L infotainment -a -G iceoryx
+    sudo usermod -L roudi -a -G iceoryx
+    sudo usermod -L notallowed -a -G iceoryx
+
+    # Allow RouDi to send SIGKILL to other apps
+    sudo setcap cap_kill=ep $WORKSPACE/build/iceoryx_examples/icecubetray/iox-cpp-roudi-static-segments
```

Comment:
@marthtz That's what I've mentioned earlier. Please check if [this capability](https://man7.org/linux/man-pages/man7/capabilities.7.html) is feasible. IMHO it can only be a workaround. QNX has a [similar way](http://www.qnx.com/developers/docs/7.1/index.html#com.qnx.doc.neutrino.lib_ref/topic/p/procmgr_ability.html) of dealing with capabilites.

Reply:
Does RouDi need this capability in general and all the time or just the special case of `iox-cpp-roudi-static-segments` or is just related to this example?
Haven't reviewed the full PR yet.

Reply:
No, this is a general requirement that was introduced with your pull request #333

Reply:
To be more precise, it is needed so that RouDi can shutdown all registered apps if they belong to a different user-id than RouDi itself. So it's a general requirement but if it is missing it won't have any impact if everything is run with the same user-id.